### ISSUE TITLE
Set global default to output excel compatible

### ISSUE THREAD

**Issue by lschaffer2** [smith-chem-wisc/MetaMorpheus#1839]
All files - psms, protein groups, peptides.

**Comment by acesnik** [comment]
Does 'excelifying' truncate columns? I'd prefer it output all data by default and not truncate. Otherwise, the data becomes unpredictable to analyze with subsequent scripts.

**Comment by zrolfs** [comment]
I agree with Anthony. We originally had MetaMorpheus truncate but then removed it for similar reasons.

**Comment by lschaffer2** [comment]
Oh gotcha. Now that I know how to change it I’m ok with that

**Comment by rmillikin** [comment]
I'll put in my vote for default behavior being Excel-compatible by default. I do understand the desire to have maximum output available. Either way the option is settable in the program, so users can pick their desired format. But in terms of new users, who may not be familiar with how to change this setting, I think it would be more user-friendly to have the data sheets easily readable in Excel

**Comment by rmillikin** [comment]
We could also have this as an option in the installer so people can just check a box when they install MM

**Comment by zrolfs** [comment]
That solution kicks the ball down the path, because we still have to decide what the default installer option should be.
I'm not sure if excel is the go-to program for visualizing data. Just sayin.

**Comment by rmillikin** [comment]
sort of, but not really.. the option is sort of hidden now. or at least not easily found if you are a new user. a checkbox in the installer is more obvious

**Comment by rmillikin** [comment]
and I can think of at least 4 MM users who use excel to interpret MM data and have been frustrated by wrapping cells (@rmillikin @lschaffer2 @blfrey RachelK). I'm extrapolating from their experiences to other less vocal users who may be confused also.

**Comment by blfrey** [comment]
I can see both sides--default or not default.  Not as elegant, but another option would be a tiny file that gets saved in the search results folder that is named "Read me if you use Excel for results".  Could be a text file, but might need to be a tsv file so that it appears if they are importing from within Excel.  This file allows you to provide more detail to the user telling them the issue, giving common examples of cell truncations, and pointing them to the user setting they can set in future runs.  It could also point them to the Excelify program, so they can correct the output file after the fact.  In a way, I like this better than making them decide at install (when they are clueless and can't really understand what checking the box/or not checking the box means).

**Comment by rmillikin** [comment]
I think a more obvious explanation and placement of this setting is certainly warranted... I can make Excelify a public program too so other people can process their results without having to re-do a search

**Comment by lschaffer2** [comment]
Could we move it from global settings to output options? Seems the most intuitive.

**Comment by acesnik** [comment]
Could we output both files as a default, so that first-time users have more options to explore the results? People with deep bioinformatics experience will want to have non-truncated columns. People who use Excel will want it to open in Excel.

@lschaffer2 Good idea! Maybe a radio box for 
1) Both full file and possibly truncated excel compatible file 
2) Just excel compatible file 
3) Just the full file

**Comment by acesnik** [comment]
The Excel-compatible PSMTSV strikes back.
![image](https://user-images.githubusercontent.com/16342951/79028611-4f88f100-7b56-11ea-9497-7e97dc2f589b.png)
